gno.land/cmd/gnoland/testdata/genesis_params.txtar

```diff
@@ -0,0 +1,14 @@
+# test for https://github.com/gnolang/gno/pull/3003
+
+gnoland start
+
+gnokey query params/vm/gno.land/r/sys/params.test.foo.string
+stdout 'data: "bar"$'
+gnokey query params/vm/gno.land/r/sys/params.test.foo.int64
+stdout 'data: "-1337"'
+gnokey query params/vm/gno.land/r/sys/params.test.foo.uint64
+stdout 'data: "42"'
+gnokey query params/vm/gno.land/r/sys/params.test.foo.bool
+stdout 'data: true'
+# XXX: bytes
```

Comment:
Leftover?

Reply:
It's because I don't know if txtar support bytes. And because I don't have any good example usage of bytes in mind too.

I can replace "XXX" with "TODO"., but Usually "XXX" means "TODO, if possible".

Reply:
@gfanton 👀